service/service.go

```diff
@@ -237,15 +243,49 @@ func (s *Service) Listen(network, address string) (net.Listener, error) {
 	return l, nil
 }
 
+func (s *Service) ListenUDP(address string) (*gonet.UDPConn, error) {
+	addr, err := net.ResolveUDPAddr("udp", address)
+	if err != nil {
+		return nil, err
+	}
+	return gonet.DialUDP(s.ipstack, &tcpip.FullAddress{
+		NIC:      nicID,
+		Addr:     tcpip.AddrFromSlice(addr.IP),
+		Port:     uint16(addr.Port),
+		LinkAddr: "",
+	}, nil, ipv4.ProtocolNumber)
+}
+
 func (s *Service) Wait() error {
-	return s.eg.Wait()
+	err := s.eg.Wait()
+
+	s.ipstack.Destroy()
```

Comment:
Adding this line made the windows test stable. They no run 200+ in a row without issue.